bin/git-commit-proxy-version

```diff
@@ -3,38 +3,48 @@
 set -eu
 
 if [ $# -ne 1 ]; then
-    echo "usage: $(basename $0) PROXY-REVISION" >&2
+    echo "usage: $(basename $0) v2.N.P" >&2
     exit 64
 fi
-new_proxy_revision="$1"
+new_proxy_version="$1"
 
 bindir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 rootdir="$( cd $bindir/.. && pwd )"
 
 old_proxy_version="$(tr -d '\n' <"$rootdir/.proxy-version")"
 if [ -z "$old_proxy_version" ]; then
-    echo "missing .proxy-version" >&2
+    echo "Missing .proxy-version" >&2
     exit 1
 fi
 
+# We temporarily have to handle aa SHA version.
+old_proxy_rev="$old_proxy_version"
+if echo "$old_proxy_version" | grep '^v2.'; then
+    old_proxy_rev="release/$old_proxy_version"
```

Comment:
If this is the older proxy revision scheme, shouldn't it be `linkerd/`, instead of `release/`?

Reply:
No, "old" here means "prior" (as in, the one we read from .proxy-version)

Reply:
Also, we'll likely continue to add linkerd2 tags on the proxy repo so it's clear how versions correlate (i.e. the tag release/v2.72.0 will likely be identical to linkerd2/edge-19.9.3)